includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1986,6 +1988,17 @@ public function is_in_test_mode() {
 		return WC_Payments::get_gateway()->is_in_test_mode();
 	}
 
+	/**
+	 * Send WC pay request using value objects.
+	 *
+	 * @param WC_Pay_Request $request - Request object.
+	 * @return Response
+	 */
+	public function send_wcpay_request( WC_Pay_Request $request ) {
```

Comment:
In one comment I asked if we plan to use Base_Request for other API than the server. Here it's quite evident that we care only about the server so `send_wcpay_server_request` would be more descriptive name.

Reply:
As mentioned in the previous comment, I will either change naming or put it into a separate folder. I am more in favor in renaming a class, but I will also consider a folder option too.